### PR TITLE
add container image list to autonotes

### PR DESCRIPTION
**Pull request by alexeldeib**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/scripts/automate_helpers.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ configure_az_devops() {
 create_branch() {
     # Create PR branch
     echo "Create branch named $1"
-    git checkout master
+    # git checkout master
```
Comment (alexeldeib) [inline vhdbuilder/scripts/automate_helpers.sh:26]: for testing only

File: vhdbuilder/packer/list-images.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
 if [[ ${CONTAINER_RUNTIME} == "containerd" ]]; then
     crictl images -o json
-    crictl images -o json | jq -c '[.images | .[] | {id:.id, repoTags:.repoTags, repoDigests:.repoDigests}]' > /opt/azure/containers/image-bom.json
+    crictl images -o json | jq '[.images | .[] | {id:.id, repoTags:.repoTags, repoDigests:.repoDigests}]' > /opt/azure/containers/image-bom.json
```
Comment (ganeshkumar5699) [inline vhdbuilder/packer/list-images.sh:8]: Do we want to store `repoDigests`? It mostly seems to be empty in the sample PR/autonotes run.

**Review by ganeshkumar5699 — APPROVED** [review]
Lgtm besides a minor comment about writing `repoDigests` field.

File: vhdbuilder/packer/list-images.sh (hunk 2 above)
Comment (alexeldeib) [reply vhdbuilder/packer/list-images.sh:8]: I swear it's sometimes non-empty, something to debug further I guess...will iterate on this, just trying to get one round of the notes out

File: vhdbuilder/scripts/automate_helpers.sh (hunk 1 above)
Comment (anujmaheshwari1) [reply vhdbuilder/scripts/automate_helpers.sh:26]: yeah lol I was gonna ask.

**Pull request merged**